meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -50,12 +55,27 @@ public ZkMetaClient(ZkMetaClientConfig config) {
 
   @Override
   public void create(String key, Object data) {
-
+    try {
+      create(key, data, EntryMode.PERSISTENT);
+    } catch (Exception e) {
+      throw new MetaClientException(e);
+    }
   }
 
   @Override
-  public void create(String key, Object data, EntryMode mode) {
-
+  public void create(String key, Object data, MetaClientInterface.EntryMode mode) {
+    int zkFlag = -1;
+    if (mode.name().equals(EntryMode.PERSISTENT.name())) {
+      zkFlag = 0;
```

Comment:
Maybe a simple translator class will be helpful.

Reply:
Sounds good! I'll make a metaClient mode to zk Mode helper function